### DISCUSSION TITLE
Merging CSV Files (with header)

### DISCUSSION THREAD

**Discussion by BillyMcSkintos** [johnkerl/miller#1039] [Q&A] [answered]
Firstly, thank you for this tool, I don't need to use it often but when I do it's great, though I am only using its most basic functions. Secondly, apologies for this basic (noob) question but I cannot figure it out.

I have a folder of TXT files delimited with a comma.
They all have the same field/header layout.
I would like to combine them into a single file with a single header.

1. Can use Miller with TXT files containing CSV?
2. Can you help with a MLR command that combines them where the output ideally has a header (from the first file perhaps), or no header?

Thank you!

**Comment by aborruso** [comment, marked as answer]
Hi @BillyMcSkintos , Miller is a really great tool.

The verbs is `cat` and starting from the folder of you TXTs, you can run

```
mlr --csv cat *.txt >output.csv
```

**Reply by BillyMcSkintos** [reply]
Thank you. That validated the command I was using but still getting the error below.
`Error: "The filename, directory name, or volume label syntax is incorrect"`
I think it's a DNS problem on my end so adding the full path of the input files resolved it.
```
